src/diffusers/models/tiny_autoencoder.py

```diff
@@ -0,0 +1,282 @@
+# Copyright 2023 Ollin Boer Bohan and The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from dataclasses import dataclass
+from typing import Callable, Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from ..configuration_utils import ConfigMixin, register_to_config
+from ..utils import BaseOutput, apply_forward_hook, is_torch_version
+from .activations import get_activation
+from .modeling_utils import ModelMixin
+from .vae import DecoderOutput
+
+
+@dataclass
+class TinyAutoencoderOutput(BaseOutput):
+    """
+    Output of TinyAutoencoder encoding method.
+
+    Args:
+        latents (`torch.Tensor`): Encoded outputs of the `Encoder`.
+
+    """
+
+    latents: torch.Tensor
+
+
+class Clamp(nn.Module):
+    def forward(self, x):
+        return torch.tanh(x / 3) * 3
+
+
+class TinyAutoencoderBlock(nn.Module):
+    def __init__(self, in_channels: int, out_channels: int, act_fn: Callable):
```

Comment:
```suggestion
    def __init__(self, in_channels: int, out_channels: int, act_fn: str):
```
I think it is better to pass a string around and only do `act_fn = get_activation(act_fn)` where you use it